### PR TITLE
[MOSIP-33034] merging  migration changes from develop-java21 to develop

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: .github/workflows/push-trigger.yml
Comment (sowmya695) [inline .github/workflows/push-trigger.yml:1]: kattu branch name change is needed

File: commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/controller/PacketReaderController.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import java.util.List;
 import java.util.Map;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
```
Comment (sowmya695) [inline commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/controller/PacketReaderController.java:8]: Remove this unused import

**Review by sowmya695 — CHANGES_REQUESTED** [review]
Please fix the review comment

File: .github/workflows/push-trigger.yml
Comment (Khuddusshariff0022) [reply .github/workflows/push-trigger.yml:1]: done

File: commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/controller/PacketReaderController.java (hunk 1 above)
Comment (Khuddusshariff0022) [reply commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/controller/PacketReaderController.java]: done

**Review by sowmya695 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**